README.md

```diff
@@ -23,7 +23,7 @@ npx: installed 1 in 1.413s
 
 you can even use `npx` to view this list of `awesome-npx` tools:
 ```
-> npx awesome-npx
+> npx awesome-npx | npx lessmd
```

Comment:
🤔 Would this be better inside the `cli.js` it self and the `package.json` as a dep?  So we could still run only `npx awesome-npx`, `npx` will take care of the deps 😛